misc/config_tools/schema/types.xsd

```diff
@@ -113,13 +113,30 @@ higher value (lower severity) are discarded.</xs:documentation>
   earliest effective virtual time. *TODO: BVT scheduler will be built on
   top of a prioritized scheduling mechanism, i.e. higher priority threads
   get scheduled first, and same priority tasks are scheduled per BVT.*
+- ``SCHED_PRIO``: The priority based scheduler. vCPU scheduling will be based on
```

Comment:
I will send a another patch to update this doc. 

This description of SCHED_BVT is OK. It may have its own prioritized mechanism which is differnt with this priority based scheduler.